### PR TITLE
Mission Hedge Maze - Minimum Impl.

### PR DESCRIPTION
**Pull request by jurcaua**
Still to do:
* Brutus Responder animations
* End of Game Cutscene?
* Cover case if we get caught right at the end sequence
* Figure out how to make nav mesh allow ai to go through the doors with ramps to outside

Work on #295

### CODE REVIEW

File: Assets/Scripts/UI/ObjectiveList/ObjectiveScriptableObjects/Kitchen1/dentures.asset
Comment (chadleytan) [inline Assets/Scripts/UI/ObjectiveList/ObjectiveScriptableObjects/Kitchen1/dentures.asset]: Aren't we acquiring Wilbur's dentures? Shouldn't this be something like "Acquire Wilbur's dentures in the kitchen"

File: Assets/Scripts/Missions/MissionKitchen1.cs
Comment (chadleytan) [inline Assets/Scripts/Missions/MissionKitchen1.cs:73]: Just to note one of the changes I made to this script involved an iteration over instantiated enemies so you may have to fix the type when you pull from master

**Review by chadleytan — COMMENTED** [review]
Beautiful work! Nice and prepped for the last mission! Left some comments on things that stood out to me

File: Assets/Scripts/UI/ObjectiveList/ObjectiveScriptableObjects/Kitchen1/dentures.asset
Comment (jurcaua) [reply Assets/Scripts/UI/ObjectiveList/ObjectiveScriptableObjects/Kitchen1/dentures.asset]: ah true yea, I'll change it to "Steal Wilbur's dentures from the kitchen"?

File: Assets/Scripts/Missions/MissionKitchen1.cs
Comment (jurcaua) [reply Assets/Scripts/Missions/MissionKitchen1.cs:73]: i have this synced with latest master, unless u have a new pr coming up soon! but yea thatll be fine :)

**Review by chadleytan — APPROVED** [review]
Okayyyy let's gooooooo!

**Review by fnnbrr — CHANGES_REQUESTED** [review]
Still needs some testing after merging from master; I'll mark as approved again once I do a test run.

**Review by fnnbrr — APPROVED** [review]

**Pull request merged**
